plugins/org.eclipse.n4js.n4mf.model/model/N4MF.xcore

```diff
@@ -25,25 +25,27 @@ package org.eclipse.n4js.n4mf
 import org.eclipse.emf.common.util.BasicEList
 import org.eclipse.n4js.utils.io.FileUtils
 
-class ProjectDescription extends SimpleProjectDescription {
+class ProjectDescription {
+	String projectId
+	String vendorId
 	String vendorName
 	contains DeclaredVersion projectVersion
 	ProjectType projectType
 	String mainModule
-	contains ExtendedRuntimeEnvironment extendedRuntimeEnvironment
-	contains ProvidedRuntimeLibraries providedRuntimeLibraries
-	contains RequiredRuntimeLibraries requiredRuntimeLibraries
-	contains ProjectDependencies projectDependencies
+	contains ProjectReference extendedRuntimeEnvironment
+	contains ProjectReference[] providedRuntimeLibraries
+	contains ProjectReference[] requiredRuntimeLibraries
```

Comment:
In the IDE, we receive a warning for `requiredRuntimeLibraries` "Obsolete required runtime libraries block." Should we remove it completely?

Reply:
not the property, but validations have to be adjusted

Reply:
please provide a sample manifest that shows the warning